mathquill4quill.js

```diff
@@ -308,15 +331,30 @@ window.mathquill4quill = function(dependencies) {
 
           historyDiv = document.createElement("div");
           let container = document.createElement("div");
-          applyHistoryContainerStyles(container);
+          applyHistoryContainerStyles(container, displayDeleteButtonOnHistory);
           let header = document.createElement("p");
           header.innerHTML = "Past formulas (max " + historySize + ")";
           historyDiv.appendChild(header);
 
           history.forEach(element => {
             const button = createHistoryButton(element, mqField);
             applyHistoryButtonStyles(button);
-            container.appendChild(button);
+            if (displayDeleteButtonOnHistory) {
+              const innerContainer = document.createElement("div");
+              applyHistoryInnerContainerStyles(innerContainer)
```

Comment:
Nit: apply `./node_modules/.bin/prettier --write "**/*.js"` so that CI passes.
```suggestion
              applyHistoryInnerContainerStyles(innerContainer);
```